tensorly/__init__.py

```diff
@@ -58,16 +58,21 @@
     ndim,
     to_numpy,
     copy,
+    float64,
     concatenate,
     reshape,
     transpose,
     moveaxis,
     arange,
+    any,
     ones,
     zeros,
     zeros_like,
     eye,
     where,
+    conj,
```

Comment:
These methods were not being exported, and I didn't see a reason for being left out.